app/views/passengers/index.haml

```diff
@@ -43,16 +45,12 @@
       - @passengers.each do |passenger|
         %tr{class: passengers_table_row_class(passenger)}
           %td= passenger.name
+          %td= passenger.mobility_device.try(:device)
```

Comment:
The passenger wouldn't have a `mobility_device`? Or is this for safety reasons?

Reply:
https://github.com/umts/st-pax/blob/3f98d3086a213d1de593a907f785ae44a5dea8a7/app/models/passenger.rb#L12

Reply:
OH